internal/test/framework/windowsvm.go

```diff
@@ -35,6 +35,9 @@ type testWindowsVM struct {
 // TestWindowsVM is the interface for interacting with a Windows VM in the test framework. This will hold the
 // specialized information related to test suite
 type TestWindowsVM interface {
+	// CopyFileTo copies the given file to the remote directory in the Windows VM. The remote directory is created if it
+	// does not exist
+	CopyFileTo(string, string) error
```

Comment:
This method is already present  in types.WindowsVM, so you shouldn't need to add it here